net/deno.json

```
@@ -4,6 +4,7 @@
   "exports": {
     ".": "./mod.ts",
     "./get-available-port": "./get_available_port.ts",
+    "./ip": "./ip.ts",
```

```suggestion
    "./unstable-ip": "./unstable_ip.ts",
```
I forgot that new modules need to have the "unstable" prefix.